packages/edit-site/src/components/layout/index.js

```diff
@@ -332,6 +341,12 @@ export default function Layout() {
 													}
 													isFullWidth={ isEditing }
 													oversizedClassName="edit-site-layout__resizable-frame-oversized"
```

Comment:
It looks like the use of this prop could be removed now and further that the prop itself could be removed from `ResizableFrame`. Of course I could've missed something.
```suggestion
```

Reply:
Good catch, thanks! 6ecff493